src/collections/posts/custom-songs.md

```diff
@@ -0,0 +1,147 @@
+
+1. In [ModAssistant](https://bsmg.wiki/pc-modding.html#mod-assistant), go to OPTIONS and click Enable OneClick Installs
+
+2. Select at least BeatSaver (though you may want the other options as well) and close ModAssistant
+
+<p align="center">
+    <img src="/uploads/mod-assistant.png">
+</p>
+
+3. On BeatSaver, find a song you want to download and click the OneClick Install button (the one that looks like a cloud with an arrow). You may see a confirmation box that your PC is trying to open ModAssistant.
+
+4. The map will be downloaded, unzipped, and added to your CustomLevels folder
+
+### Custom Songs with the Base Game (More difficult!)
+
+BeastSaber Download Button
+
+1. On BeatSaver, find a song you want to download and click Download Zip and a zip file of the song you want will be saved to your computer
+
+2. Extract the zip file as a folder. It should already be titled with the map's "key" or the unique identifier for the map, and the song name and mapper(s) in parenthesis
+
+> 🗒️ NOTE that in the BeatSaver URL, you'll see beatsaver.com/maps/**KEY** - So you can always come back to the map page if you need to
+
+  - You can either extract in place and then cut/paste the folder into your CustomLevels folder
+
+  - Or you can select the path directly to your Custom Levels folder and extract it directly there.
+
+Oculus User folder paths tend to be: `C:\Program Files\Oculus\Software\Software\hyperbolic-magnetism-beat-saber\Beat Saber_Data\CustomLevels`
+
+Steam User folder paths tend to be: `C:\Program Files (x86)\Steam\steamapps\common\Beat Saber\Beat Saber_Data\CustomLevels`
+
+<br />
+
+## Custom Songs for Quest VR Users
+Getting custom songs on the Quest requires a little more work than with its PC cousins. There are two pieces of software you’ll need: [SideQuest](https://sidequestvr.com/setup-howto) and [BMBF](https://bmbf.dev/stable) or [QuestPatcher](https://github.com/Lauriethefish/QuestPatcher/releases).
+
+> **💡 HINT: When downloading SideQuest, be sure to use the "Advanced Installer". This will greatly help you when adding songs!**
+
+Once you have the right software installed, you have a couple of options on how to access songs. The [Beat Saber Modding Group Wiki](https://bsmg.wiki/quest-modding.html#installing-songs) is the best place to get the latest information, but here’s a quick step-by-step:
+
+1. Make sure you have the latest BeatSaber mods available for your moddable game. Keep in mind that Quest modding is usually well behind PC modding so you will likely need to downgrade. You can find the latest mods by clicking [here](https://computerelite.github.io/tools/Beat_Saber/questmods.html)
+
+2. Using QuestPatcher, click Browse and open the mod pack you downloaded from the link above
```

Comment:
It seems a bit inconsistent. Here you don't quote "Browser" but you quote "MORE SONGS" under 3.